src/Cosi-BLS/cosi-handlers.lisp

```diff
@@ -775,6 +775,57 @@ check that each TXIN and TXOUT is mathematically sound."
         (wait))
       )))
 
+;; ------------------------------
+
+(defun gossip-neighborcast (node &rest msg)
+  "Gossip-neighborcast - send message to all nodes. Just a stub for now."
+  (declare (ignore node msg))
+  t)
+
+(=defun gossip-signing (node my-ip consensus-stage msg seq-id)
```

Comment:
@dbmcclain Doesn't the `my-ip` argument means that pseudo-nodes need to have to have a randomly generated ip address associated with them somehow.  I thought that this wasn't necessarily the case now.

Reply:
Nothing has changed in the simulation structure of the Cosi nodes yet. Still in preparation for merging with Gossip protocol. This new code simply provides a test of the ability to retain tree-like and/or gossip-like communications in Cosi signature gathering.

Reply:
The my-ip argument in this case refers to the leaders actual IP for use as a reply-to argument. So, actually, this is orthogonal to the issue of simulation. We need to tell each witness node who to send their partial signatures to. For now, I'm having the leader do all the coalescing.